bugsnag-android-core/src/main/java/com/bugsnag/android/DefaultDelivery.kt

```diff
@@ -1,14 +1,16 @@
 package com.bugsnag.android
 
-import java.io.BufferedWriter
 import java.io.IOException
+import java.io.OutputStream
 import java.io.OutputStreamWriter
 import java.net.HttpURLConnection
 import java.net.HttpURLConnection.HTTP_BAD_REQUEST
 import java.net.HttpURLConnection.HTTP_CLIENT_TIMEOUT
 import java.net.HttpURLConnection.HTTP_OK
 import java.net.URL
 import java.nio.charset.Charset
+import java.security.DigestOutputStream
+import java.security.MessageDigest
```

Comment:
These classes don't seem to be used in this file?

Reply:
Removed unused imports